### ISSUE TITLE
[Blazor WASM] Breakpoints not being hit in Visual Studio

### ISSUE THREAD

**Issue by aventus13** [dotnet/aspnetcore#22179]
### Describe the bug
When attempting to debug a basic counter page in Visual Studio following steps [from this article](https://docs.microsoft.com/en-us/aspnet/core/blazor/debug?view=aspnetcore-3.1#visual-studio), the breakpoint is not getting hit as "no symbols have been loaded".

I was able to enable debugging in Chrome, and debug the code in browser's dev tools. Attempting to debug the application in Visual Studio was unsuccessful.

### To Reproduce
- Using Visual Studio create a new Blazor WebAssembly project _without_ HTTPS and _with_ ASP.Net Core hosted
- The solution should be created with three projects: x.Client, x.Server, and x.Shared
- Open _Counter.razor_ file in the Client project and set a breakpoint on `currentCount++` line
- Start the Server application by hitting F5 and observe the breakpoint
- The breakpoint will be disabled with message "The breakpoint will not currently be hit. No symbols have been loaded for this document."
- Use the application in the browser- navigate to Counter page and click "Click me" button
- The UI will update correctly however the breakpoint will not get hit in Visual Studio

### Further technical details
- ASP.NET Core 3.1
- Visual Studio 2019 16.5.5
- Microsoft.AspNetCore.Components.WebAssembly.Templates 3.2.0

**Comment by snerks** [comment]
In my experience, it takes a relatively long time for the debugger to attach.

Have you tried the following?:
1. Launch the application under Debug
1. Wait for (approximately) 10 seconds
1. While waiting, watch the Visual Studio breakpoints, until they switch to red blobs
1. Switch from/to the Counter tab

I found that, at this point, breakpoints work as expected.

**Comment by aventus13** [comment]
Hi @snerks, thanks for you feedback. 

Yes I have tried waiting, switching between pages etc. Just to be clear- I can still debug the server's code just fine, for example setting a breakpoint in `WeatherForecastController` is working as expected. It looks like the only thing that doesn't work are the breakpoints set in razor components. I also tried debugging a standalone Blazor WebAssembly app (no Server, just the Client project) to no avail. 

I'm unable to hit breakpoints in razor components no matter what I try. I've been playing around with different port numbers, starting the app with F5 keys as well as right clicking the project > Debug > Start new instance, switching between Chrome and Edge. Getting same results every time.

Just in case this is also relevant, I'm using the following version of Google Chrome:
Version 83.0.4103.61 (Official Build) (64-bit)
